test/integration/api.bats

```diff
@@ -12,6 +12,26 @@ function teardown() {
 	skip
 }
 
+@test "docker attach through websocket" {
+	CLIENT_API_VERSION="v1.17"
```

Comment:
We should probably expose this at a higher level, in `helpers.bats`? /cc @aluzzardi